packages/oauth/oauth-provider/src/token/token-manager.ts

```diff
@@ -546,9 +523,6 @@ export class TokenManager {
 
   protected async findTokenInfo(token: string): Promise<TokenInfo | null> {
     switch (true) {
-      case isTokenId(token):
-        return this.store.readToken(token)
-
```

Comment:
This case looks correct at a glance, and it makes the method comprehensive.  Was this removed because there's no code path that hits it?  It is just a little hard to tell whether there might be such a codepath when the types can't back it up in this case.

Reply:
This code is used through OAuth's token introspection endpoint, which is supposed to only be called with either an access token or a refresh token. Since access tokens are always JWTs, we do not need to support introspecting a token id, besides for legacy reasons.

I've added the behavior back (for legacy)

Reply:
Yeah I'd either add the behavior back or just leave a comment explaining that.